tests/performance/test_data_analysis/data_analyzer_module/test_data_analyzer_module.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""
+Statistical Data Analyzer Module Test
```

Comment:
```suggestion
"""Statistical Data Analyzer Module Test.
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3